Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.ts

```diff
@@ -240,6 +240,7 @@ class BocAutoCompleteReferenceValue //TODO RM-7715 - Make the TypeScript classes
         }
 
         const hasChanged = this._itemBackUp != actualItem;
+        const isEmpty = actualItem.DisplayName === '';
```

Comment:
I would move this towards the usage below as we don't use it multiple times and it makes it harder to track the value of `isEmpty`